### DISCUSSION TITLE
Any way to create or list database connections from the CLI?

### DISCUSSION THREAD

**Discussion by zblox-project** [auth0/auth0-cli#1210] [General]
### Describe the problem you'd like to have solved

It seems I should be able to create a new database connection from CLI. Maybe I am missing details in the documentation?

### Describe the ideal solution

auth0 dbconnection create
auth0 dbconnection list
auth0 dbconnection delete

## Alternatives and current work-arounds

na

### Additional context

na

**Comment by Widcket** [comment]
Hi @stephan-erickson, thanks for raising this. The Auth0 CLI does not currently support database connection CRUD.

**Comment by mstallmo** [comment]
Just wanted to come and give this issue a bump for visibility. Adding this feature to the CLI would be very helpful!

Would a PR from the community implementing this feature be accepted?

**Comment by Widcket** [comment]
cc @sergiughf ^

**Comment by sergiught** [comment]
Hey @mstallmo, @stephan-erickson, appreciate you folks raising visibility around this. We don't currently plan on adding this feature to the cli right now, but contributions are more than welcome. We'll review them together and collaborate on getting them to the finish line:) so please feel free to give it a try.

**Comment by sergiught** [comment]
Hey folks 👋🏻 ,

We recently added a new command 🧰 `auth0 api <method> <url-path> [flags]` that is able to make authenticated requests against the management API (https://auth0.com/docs/api/management/v2). 

:octocat: This command is available with https://github.com/auth0/auth0-cli/releases/tag/v0.13.0. 

📖 Docs for this command: https://auth0.github.io/auth0-cli/auth0_api.html

Although it doesn't offer first class support for managing database connections, you could use it as an alternative to manage database connections through the cli until we have a more dedicated command for them.

For example to create a database connection: `auth0 api post connections  --data "{\"strategy\":\"auth0\", etc...}"`

Feedback is much appreciate for this new command 🙏🏻 Let us know if you encounter any issues with it. 

We'll keep this issue open however until we are able to add a dedicated first class command to manage db connections. 

Thanks for your patience! ❤️
